uitools/import/Esri/ArcGISRuntime/Toolkit/SearchView.qml

```diff
@@ -19,7 +19,7 @@ import Esri.ArcGISRuntime.Toolkit.Controller
 import QtQuick 2.12
 import QtQuick.Controls 2.12
```

Comment:
please log a follow up issue to remove all QML module versions from the imports

Reply:
Done.